tests/cli.rs

```diff
@@ -0,0 +1,160 @@
+use assert_cmd::assert::IntoOutputPredicate;
```

Comment:
I really appreciate the cli tests!

Reply:
Glad to help! I'm sure these can be much improved upon after further tinkering and figuring out how to do this well. These are a bit blunt and brittle since they rely so much on the text I/O, but it should at least be a good starting point for some end-to-end style tests.